src/Tizen.Log/Tizen/Log.cs

```diff
@@ -112,17 +114,52 @@ public static void Fatal(string tag, string message, [CallerFilePath] string fil
             Print(Interop.Dlog.LogPriority.DLOG_FATAL, tag, message, file, func, line);
         }
 
-        static void Print(Interop.Dlog.LogPriority priority, string tag, string message, string file, string func, int line)
+        static unsafe void Print(Interop.Dlog.LogPriority priority, string tag, string message, string file, string func, int line)
         {
-            if (String.IsNullOrEmpty(file))
-            {
-                Interop.Dlog.Print(priority, tag, "%s", message);
-            }
-            else
+            int tagByteLength = Encoding.UTF8.GetMaxByteCount(tag.Length);
+            Span<byte> tagByte = tagByteLength < 1024 ? stackalloc byte[tagByteLength + 1] : new byte[tagByteLength + 1];
+
+            int messageByteLength = Encoding.UTF8.GetMaxByteCount(message.Length);
+            Span<byte> messageByte = messageByteLength < 1024 ? stackalloc byte[messageByteLength + 1] : new byte[messageByteLength + 1];
+
+            fixed (char* pTagChar = &MemoryMarshal.GetReference(tag.AsSpan()))
+            fixed (char* pMessageChar = &MemoryMarshal.GetReference(message.AsSpan()))
+            fixed (byte* pTagByte = &MemoryMarshal.GetReference(tagByte))
+            fixed (byte* pMessageByte = &MemoryMarshal.GetReference(messageByte))
             {
-                int index = file.LastIndexOfAny(sep);
-                string filename = file.Substring(index + 1);
-                Interop.Dlog.Print(priority, tag, "%s: %s(%d) > %s", filename, func, line, message);
+                int len = Encoding.UTF8.GetBytes(pTagChar, tag.Length, pTagByte, tagByteLength);
+                pTagByte[len] = 0;
+                len = Encoding.UTF8.GetBytes(pMessageChar, message.Length, pMessageByte, messageByteLength);
+                pMessageByte[len] = 0;
+
+                if (String.IsNullOrEmpty(file))
+                {
+                    Interop.Dlog.Print(priority, pTagByte, "%s", pMessageByte);
+                }
```

Comment:
nit: return here and remove `else` clause for readability.

Reply:
Okay, I'll fix it. At first, I will upload the pull request again after "[[Tizen.Log] Optimize garbage collection]" this pull request. My code is so dependent on that pull request. Thus I will upload the patch again. Thanks for your reviews.